Chp9Lsn11/Chp9Lsn11/Program.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using UserInputHandler;
+
+namespace Chp9Lsn11
+{
+    public class Program
+    {
+        static void Main(string[] args)
+        {
+            string answer = "Error";
+            string userCommand = TextMenu(answer);
+            MultipleType FinalAnswer = CallOption(userCommand);
+            PrintAnswer(userCommand, FinalAnswer);
+            Console.ReadLine();
+        }
+
+        public static string TextMenu(string answer)
+        {
+            bool resolved = false;
```

Comment:
This violates "Design by Contract".  The `TextMenu` method needs to have two variables initialized to work correctly (`resolved` and `answer`).  Currently `TextMenu` depends on some external caller / customer / client to provide a parameter equal to "Error" in order for `TextMenu` to initialize correctly.